com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 - Fixed NetworkVariable not calling NetworkSerialize on INetworkSerializable types (#1383)
 
 - Fixed NetworkObjects not being despawned before they are destroyed during shutdown for client, host, and server instances.
```

Comment:
also why this entry doesn't have a PR number attached? 👀

Reply:
Fixed.